### ISSUE TITLE
System.IO.IOException: The file cannot be accessed by the system.

### ISSUE THREAD

**Issue by Kajanor** [ShareX/ShareX#2664]
After uninstalling a Windows Insider Preview Build and going back to the "stable" version of Windows, ShareX now gives me this error on program startup:
```
System.IO.IOException: The file cannot be accessed by the system.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
   at ShareX.Program.ReadPersonalPathConfig()
   at ShareX.Program.CheckPersonalPathConfig()
   at ShareX.Program.UpdatePersonalPath()
   at ShareX.Program.Main(String[] args)

ShareX 11.8 (Release)
```

I already tried reinstalling the program but that didn't work.

**Comment by kirhgoph** [comment]
Are you able to open and move the file located at **C:\Users\<your_name>\Documents\ShareX\PersonalPath.cfg** ?

**Comment by Kajanor** [comment]
I don't know exactly what happened to the files, I only think of Windows might have corrupted them during the downgrade, I also noticed the screenshots I took were all unreadable.
Nevertheless I could "solve" the issue by deleting the ShareX folder in the Documents directory and initializing the program afterwards.
Now ShareX is working as intended and my new screenshots aren't corrupted files any more.
